paasta_tools/cli/cmds/validate.py

```diff
@@ -604,6 +604,14 @@ def validate_autoscaling_configs(service_path):
                                 link="",
                             )
                         )
+                    if len(instance_config.get_registrations()) > 1:
```

Comment:
comment for future yelpers looking at this PR: should we ever fix `get_registrations()` to not return a value for non-mesh-routable services, we'd probably also want to add validation blocking autoscaling said services with this provider :)